### PR TITLE
Better: Add documentation about Huawei Push Notifications for Engage Messaging. RD-18728

### PR DESCRIPTION
**Pull request by DrissTM**
Hey!

**Ticket(s)**:
RD-18728

**Change(s)**:
- Add documentation about the Huawei Push Notifications configuration for Engage Digital Messaging channel

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -34,6 +34,8 @@ pages:
   - 'Interactions':
     - 'Overview': interactions/index.md
     - 'Getting Started': interactions/quick-start.md
+    - 'Mobile Messaging':
```
Comment (CedricBm) [inline mkdocs.yml:37]: I think the Mobile Messaging part should be after the web messaging, no?

**Review by CedricBm — CHANGES_REQUESTED** [review]
Nice, just an ordering suggestion, then looks good to me.

File: mkdocs.yml (hunk 1 above)
Comment (DrissTM) [reply mkdocs.yml]: no strong opinion on this, I only chose this order because messaging sources were originally mobile only

changed in 66d964d

**Comment by CedricBm** [comment]
LGTM

**Review by CedricBm — APPROVED** [review]

**Comment by DrissTM** [comment]
merged manually

**Pull request closed without merging**